tests/foreman/cli/test_syncplan.py

```diff
@@ -697,7 +695,7 @@ def test_positive_synchronize_rh_product_future_sync_date(
         }
     )
     sync_plan = entities.SyncPlan(organization=org.id, id=new_sync_plan['id']).read()
-    request.addfinalizer(lambda: disable_syncplan(sync_plan))
+    request.addfinalizer(lambda: target_sat.api_factory.disable_syncplan(sync_plan))
     # Verify product is not synced and doesn't have any content
     with pytest.raises(AssertionError):
         validate_task_status(repo['id'], org.id, max_tries=1)
```

Comment:
A Satellite object should be passed to the `validate_task_status` function after your change. Could you please check that all callers of `validate_task_status` are adding the `sat` argument to it? It would be a good idea to check it across PRs/branches.

Reply:
Nice Catch! Did change in this and 6.11 PR, I have taken reference from master and master, 6.13 and 6.14 are in sync as the changes are cherrypicked. 

So now we should be good !